plugins/inlinetoolbar/plugin.js

```diff
@@ -55,7 +55,7 @@
 	 *				var lastElement = evt.data.path.lastElement;
 	 *
 	 *				if ( lastElement ) {
-	 *					toolbar.create( lastElement );
+	 *					toolbar.attac( lastElement );
```

Comment:
I mean... c'mon 😉

Reply:
mea culpa. I don't know why, I thought that I missed `create` somewhere... And actually I missed one in example. And forgot about typo.